url-updater/src/main/java/com/devonfw/tools/ide/url/updater/gcloud/GCloudUrlUpdater.java

```diff
@@ -0,0 +1,46 @@
+package com.devonfw.tools.ide.url.updater.gcloud;
+
+import com.devonfw.tools.ide.url.model.folder.UrlVersion;
+import com.devonfw.tools.ide.url.updater.WebsiteUrlUpdater;
+
+import java.util.regex.Pattern;
+
+public class GCloudUrlUpdater extends WebsiteUrlUpdater {
+
+  @Override
+  protected String getTool() {
+
+    return "gcloud";
+  }
+
+  @Override
+  protected void addVersion(UrlVersion urlVersion) {
+
+    String baseUrl = "https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-${version}-${os}-";
+
+    doAddVersion(urlVersion, baseUrl + "x86_64.zip", WINDOWS);
+    //TODO add Linux and MacOS
+  }
+
+  @Override
+  protected String getVersionUrl() {
+
+    return "https://cloud.google.com/sdk/docs/release-notes";
```

Comment:
our goal is to get rid of `WebsiteUrlUpdater` and only use it as last resort.
Please consider using `GitHubUrlUpdater` instead:
https://github.com/twistedpair/google-cloud-sdk/tags